### PR TITLE
Update option shown in `Vv` when seeking to other symbols in `V`

### LINKED ISSUE #2751: `v` mode starts from the offset entered before going to `V` mode, not the current selected function
### Work environment

| Questions                                            | Answers
|------------------------------------------------------|--------------------
| OS/arch/bits (mandatory)                             | MacOS Monterey ARM64
| File format of the file you reverse (mandatory)      | MachO
| Architecture/bits of the file (mandatory)            | ARM64
| `rizin -v` full output, **not truncated** (mandatory)  | https://github.com/rizinorg/rizin/commit/2987e035da827903329ec6433410899b985e2434

### Steps to reproduce the behavior
1. `rizin /bin/ls`
2. `aaa`
3. Open visual mode with `V`
4. Go to some symbol using `g`: `sym.printf`
5. Open visual menu mode with `v`
6. It goes to the entry point (or any offset that was set before going to the `V` mode (point 3))

It should open with `sym.printf` selected already in the `v` mode.

### PR DESCRIPTION
**Pull request by theopechli**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

After analyzing a binary, when the user:

1. entered visual mode with `V`
2. used `g` to go to a symbol
3. and entered `v`

then the current option would not be set to the location of the symbol the user seeked at.

**Test plan**

**Closing issues**

closes #2751

### CODE REVIEW

**Review by XVilka — CHANGES_REQUESTED** [review]
Doesn't fix the problem completely. While the right flag is selected in the "roster" on the left side, the right side still shows `entry0` contents.

**Review by XVilka — APPROVED** [review]
It works properly now, thanks! Lets wait for one more review first though.

**Review by ret2libc — APPROVED** [review]
Works!

**Pull request merged**
